nuxeo-core/nuxeo-core-schema/src/main/java/org/nuxeo/ecm/core/schema/FacetNames.java

```diff
@@ -105,6 +105,14 @@
      */
     public static final String HAS_RELATED_TEXT = "HasRelatedText";
 
+    /**
+     * Facet used to indicate that a given document can have his content stored as a cold storage, this is the case for
+     * example where the content represents: archiving, compliance...
```

Comment:
Why not keep the pattern used for the other facets?
Something like: "The document can have its main content stored as cold storage. This can be needed for archiving, compliance, etc."

Reply:
yep, on some facet they used the pattern `Facet to be used fo.....` for example `HAS_RELATED_TEXT`  this is the reason why i changed it :), but yes why not i will make the modification ;)